app/views/admin/conferences/new.html.haml

```diff
@@ -9,7 +9,7 @@
                                 input_html: { required: 'required', pattern: '[a-zA-Z0-9_-]+', title: 'Only letters, numbers, underscores, and dashes.' }, prepend: conferences_url + '/'
       = f.inputs 'Scheduling' do
         = f.input :timezone, as: :time_zone, default: Time.zone.name, hint: 'Please select in what time zone your conference will take place.'
-        = f.input :start_date, as: :string, input_html: { id: 'conference-start-datepicker', required: 'required' }
-        = f.input :end_date, as: :string, input_html: { id: 'conference-end-datepicker', required: 'required' }
+        = f.input :start_date, as: :string, input_html: { id: 'conference-start-datepicker', required: 'required', readonly: 'readonly' }
```

Comment:
why do you set readonly?

Reply:
To prevent user to input manually a date. And to be coherent with the first form I encountered (Registration Period) that was already read only.
To insure however that the datetime picker can update with a readonly attribute in html, I had to add an ignoreReadonly in the datepicker js file. (See my comment above).

Reply:
I wouldn't prevent the user to manually input a date. I would make it coherent the other way around.

Reply:
Ok, I will update the changes in that way and remove the readonly attributes.

Reply:
I have pushed the changes. I hope this is ok.